proto/runtime.proto

```diff
@@ -31,7 +31,8 @@ message StartResponse {
   // Was the start successful
   bool success = 1;
 
+  // todo: find a way to add optional flag here
   // Optional port the service was started on
   // This is likely to be None for bots
-  optional uint32 port = 2;
```

Comment:
This was causing an error about optional not being allowed in proto3 syntax, even though it should be in the versions we use. Ref: https://github.com/bufbuild/buf/issues/275

This meant I also ended up adding a temp hardcoded port to the `next` runtime bot.